api/src/main/resources/messages.properties

```diff
@@ -642,7 +642,7 @@ coreapps.conditionui.updateCondition.success=Condition Saved Successfully
 coreapps.conditionui.updateCondition.error=Error Saving condition
 coreapps.conditionui.removeCondition=Remove Condition
 coreapps.conditionui.removeCondition.message=Are you sure you want to remove {0} condition for this patient?
-coreapps.conditionui.editCondition=Edit Condition: {0}
```

Comment:
Why are we removing this parameter {0}?

Reply:
I felt  the heading looked good if it was just Edit Condition , rather than having Edit Condition : {some condition}  as the condition field is being displayed right below the heading. What do you think of it?

Reply:
Can you share that with Ellen and the rest?

Reply:
sure :)